### PR TITLE
Update StandardToken.sol

### PR DESCRIPTION
**Pull request by pipaman**
Added increaseApproval and decreaseApproval to increase / decrease the approval in 1 transaction.

### CODE REVIEW

**Comment by maraoz** [comment]
I like this approach. Thanks for the contribution
Would you mind adding some tests?

**Comment by maraoz** [comment]
@pipaman ping on adding tests ;)

File: contracts/token/StandardToken.sol [hunk 1]
```diff
@@ -43,5 +43,32 @@ contract StandardToken is BasicToken, ERC20 {
...
   }
+  
+    /*
+   * approve should be called when allowed[_spender] == 0. To increment
+   * allowed value is better to use this function to avoid 2 calls (and wait until 
+   * the first transaction is mined)
+   * From MonolithDAO Token.sol
+   */
+  function increaseApproval (address _spender, uint _addedValue) 
+    onlyPayloadSize(2 * 32)
```
Comment (rudygodoy) [inline contracts/token/StandardToken.sol]: Never mind the previous comment about dependencies. I'm working on this.

Comment (frangio) [inline contracts/token/StandardToken.sol]: `onlyPayloadSize` was removed (#277). This PR needs rebasing.

**Comment by ajsantander** [comment]
@pipaman ping ping

**Comment by facuspagnuolo** [comment]
Closing since it is already implemented

**Pull request closed without merging**